sklearn/utils/_repr_html/estimator.js

```diff
@@ -32,7 +32,7 @@ function copyToClipboard(text, element) {
     return false;
 }
 
-document.querySelectorAll('.fa-regular.fa-copy').forEach(function(element) {
+document.querySelectorAll('.copy-paste-icon').forEach(function(element) {
```

Comment:
This makes the hover over the copy icon message work again.
It broke on the first HTML Display PR when the icon was changed from a download to `svg` - but didn't notice as we don't have a test for it. :(